playground/src/components/VarItem.vue

```diff
@@ -63,7 +70,7 @@ function onclick() {
     >
     <input
       v-if="type === 'color'"
-      v-model="value"
+      v-model="color"
```

Comment:
If we bound v-model with a computed, it will make all color picker readonly

Reply:
updated.